### PR TITLE
fix: change confirm button text to 'yes, im sure'

### LINKED ISSUE #1705: better to improve text of are-you-sure-to-delete-your-data  modal
### Describe the bug:
Currently modal has too many repetitive text:

Screen Shot 2022-10-13 at 12 41 06

I think 'confirm' button better to have 'Yes, im sure!' text

### Tell us about your browser and operating system:
- Browser(s) name and version: chrome
- Operating System: mac

### PR DESCRIPTION
**Pull request by shootermv**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #1705

### CODE REVIEW

File: client/src/modules/profiles/pages/userProfile.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const UserProfilePage = () => {
   const logout = useLogout();
   const router = useRouter();
 
-  const confirmDelete = useConfirmDelete({ doubleConfirm: true });
+  const confirmDelete = useConfirmDelete({ buttonText: 'Yes, im sure' });
```
Comment (Sboonny) [inline client/src/modules/profiles/pages/userProfile.tsx:24]: I don't know what happens but after deleting the using, 

auth with google is deleted

image

---

I have no issue with it, just wondering if we okay with single confirm?

Why did we have double confirm before?

**Review by Sboonny — CHANGES_REQUESTED** [review]
this isn't about the file itself, but there is similar case in ChaptePage in the dashboard

here is the code

```ts
const clickDelete = async () => {
    const ok = await confirmDelete({ doubleConfirm: true });
    if (!ok) return;
    deleteChapter({ variables: { chapterId } });
    router.push('/dashboard/chapters');
  };
  ```
  if you change it too, that would be lovely 👍

File: client/src/modules/profiles/pages/userProfile.tsx (hunk 1 above)
Comment (gikf) [inline client/src/modules/profiles/pages/userProfile.tsx:24]:
```suggestion
  const confirmDelete = useConfirmDelete({
    body: 'Are you sure you want to delete your account? Account deletion cannot be reversed.',
    buttonText: 'Delete account',
  });
```
Could we change at the same time the default text? And maybe also make explicit that is account what is being deleted.

Comment (gikf) [reply client/src/modules/profiles/pages/userProfile.tsx:24]: Is that instance owner logged in?

Comment (Sboonny) [reply client/src/modules/profiles/pages/userProfile.tsx:24]: nope I deleted the instance owner 🤣

Comment (gikf) [reply client/src/modules/profiles/pages/userProfile.tsx:24]: Exactly :slightly_smiling_face:

**Review by Sboonny — APPROVED** [review]

**Pull request merged**